pkg/controller/pod.go

```diff
@@ -347,7 +347,10 @@ func (pnc *PodNetworksController) updatePodNetworkStatus(pod *corev1.Pod, newIfa
 func networkSelectionElements(podAnnotations map[string]string, podNamespace string) ([]*nadv1.NetworkSelectionElement, error) {
 	podNetworks, ok := podAnnotations[nadv1.NetworkAttachmentAnnot]
 	if !ok {
-		return nil, fmt.Errorf("the pod is missing the \"%s\" annotation on its annotations: %+v", nadv1.NetworkAttachmentAnnot, podAnnotations)
+		return []*nadv1.NetworkSelectionElement{}, nil
```

Comment:
I think this could be simplified by using

```go
if !ok || podNetworks == "" {
   return []*nadv1.NetworkSelectionElement{}, nil
}
```

Reply:
Done.